Assets/Scripts/Editor/DefaultSceneLoader.cs

```diff
@@ -22,7 +23,7 @@ static void LoadDefaultScene(PlayModeStateChange state)
                 EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo();
             }
 
-            if (state == PlayModeStateChange.EnteredPlayMode)
+            if (state == PlayModeStateChange.EnteredPlayMode && SceneManager.GetActiveScene().buildIndex != 0)
```

Comment:
Oh oeps? Zat er een loop in? :D

Reply:
nee, maar als je al in de config scene zat ging hij alsnog de config scene laden, dat leverde wat problemen op met het inladen van het default project toen ik eraan werkte, volgens mij moet het nu niet meer een probleem zijn, maar alsnog wel goed om dit te laten staan denk ik